examples/room_number/custom/intents.json

```diff
@@ -1,7 +1,7 @@
 {
   "intents": [
     {
-      "label": "calling_room",
+      "label": "room_number",
```

Comment:
I agree with @AGiantSquid of renaming the folder instead. It is confusing to have the entity and intent named the same, and we do not want to promote this pattern